### PR TITLE
Add Postgres

### PR DESCRIPTION
**Pull request by thewilkybarkid**
https://github.com/libero/publisher/issues/358

### CODE REVIEW

File: helm/publisher/values.yaml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 images:
   articleStore:
     repository: liberoadmin/article-store
-  pullPolicy: IfNotPresent
+
+postgresql:
+  nameOverride: article-store-postgresql
+  persistence:
+    enabled: false
+  postgresqlPassword: pAsSw0rD
```
Comment (thewilkybarkid) [inline helm/publisher/values.yaml:9]: Default chart behaviour is to update the secret on every release, but the change is not propagated. So, future app pods get the new version, but the Postgres pod _doesn't_ (unless it happens to be recreated on the same release _and_ has no persistence).

tl;dr: set secret values manually when using Helm

Refs https://github.com/helm/charts/issues/5167

File: helm/publisher/templates/article-store-deployment.yaml [hunk 2]
```diff
@@ -19,7 +19,20 @@ spec:
       containers:
         - name: app
           image: "{{ .Values.images.articleStore.repository }}:latest"
-          imagePullPolicy: {{ .Values.images.pullPolicy }}
```
Comment (thewilkybarkid) [inline helm/publisher/templates/article-store-deployment.yaml:22]: `IfNotPresent ` conflicts with the `latest` tag, as it never updates.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -26,6 +26,7 @@ jobs:
             quay.io/helmpack/chart-testing:v3.0.0-beta.1 \
             ct lint \
             --chart-dirs=helm \
+            --chart-repos stable=https://kubernetes-charts.storage.googleapis.com/ \
```
Comment (thewilkybarkid) [inline .github/workflows/ci.yml:29]: Annoying, but hopefully could be resolved by https://github.com/helm/helm/pull/7242 in the future.

File: README.md [hunk 4]
```diff
@@ -9,7 +9,6 @@ This repository contains a functional Helm chart for the deployment of Libero Pu
 The product is set up with a vanilla configuration:
 
 - a single `article-store`
-- an Hypermedia console
```
Comment (thewilkybarkid) [inline README.md:12]: Unrelated, but isn't true.

**Review by giorgiosironi — APPROVED** [review]

**Pull request merged**